docs/sdks/node-cli/GETTING_STARTED.md

```diff
@@ -0,0 +1,156 @@
+## Getting Started 
+
+Before you can use the CLI, you need to login to your Appwrite account. 
+
+```sh
+$ appwrite login
+
+? Enter your email test@test.com
+? Enter your password ********
+✓ Success 
+```
+This will also prompt you to enter your appwrite endpoint ( default: http://localhost/v1 ) 
+
+* ### Initialising your project
+Once logged in, the CLI needs to be initialised before you can use it with your Appwrite project. You can do this with the `appwrite init project` command. 
+
+```sh
+$ appwrite init project
+```
+
+The following prompt will guide you through the setup process. The `init` command also creates an `appwrite.json` file representing your Appwrite project.
+
+The `appwrite.json` file does a lot of things. 
+* Provides context to the CLI
+* Keeps track of all your cloud functions
+* Keeps track of all your project's collections
+* Helps you deploy your Appwrite project to production and more..
+
+You can also fetch all the collections in your current project using
+```sh
+appwrite init collection
+```
+
+The CLI also comes with a convenient `--all` flag to perform both these steps at once using
```

Comment:
```suggestion
The CLI also comes with a convenient `--all` flag to perform both these steps at once.
```

Reply:
I've made this suggestion because the instruction on the `--all` flag for the `deploy` command is also mentioned in the suggested format.